src/com/projectkorra/projectkorra/board/BendingBoard.java

```diff
@@ -61,7 +65,13 @@ public void setSlot(int slot) {
 		}
 		
 		public void decreaseSlot() {
-			setSlot(--slot);
+			setSlot(slot - 2);
+			String prefix = team.getPrefix(), suffix = team.getSuffix();
+			this.board.resetScores(entry);
+			this.team.unregister();
+			this.formTeam();
+			this.update(prefix, suffix);
+			this.set();
```

Comment:
Is this call to `set()` necessary? If I understand correctly, the prefix and suffix are being saved so we can reuse them after the call to unregister but `update(...)` will also call `set()`.

Reply:
u right dawg